internal/component/database_observability/mysql/component.go

```diff
@@ -213,6 +213,7 @@ func enableOrDisableCollectors(a Arguments) map[string]bool {
 	collectors := map[string]bool{
 		collector.QueryTablesName: true,
 		collector.SchemaTableName: true,
+		collector.QuerySampleName: true,
```

Comment:
Do we also need to add the collector in startCollectors below? line 233?
Also the existing Error log messages for the QueryTables collector say "QuerySample".

Reply:
Fixed in b51971003bad48edcc7d1831303875f83d6cbe01